src/software/visualizer/drawing/BUILD

```diff
@@ -76,13 +92,31 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "obstacle_artist",
```

Comment:
super nit: I think just naming this `obstacle` is more consistent with the other targets / files here

Reply:
I think the library name should be consistent with the file name and class name

Reply:
yea I guess by extension those are inconsistent too, but this isn't blocking

Reply:
I'm not going to change this then